src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java

```diff
@@ -21,9 +22,11 @@
 
     private final UserRoleRepositoryService service;
 
+    @Value("${spring.servlet.multipart.max-file-size}")
```

Comment:
why is this not part of Configuration?

Reply:
TermIt configuration contains only TermIt-related configuration, nothing Spring related.